cylc/flow/task_state.py

```diff
@@ -527,3 +527,12 @@ def get_unsatisfied_prerequisites(self) -> List['PrereqMessage']:
             for prereq in self.prerequisites if not prereq.is_satisfied()
             for key, satisfied in prereq.items() if not satisfied
         ]
+
+    def any_satisfied_prerequisite_tasks(self) -> bool:
+        """Return True if any of this task's prerequisite tasks are
+        satisfied."""
```

Comment:
I am using the term "prerequisite tasks" to avoid confusion with `Prerequisite`s which can comprise multiple prerequisite tasks.

Reply:
Try `Prerequisite outputs".

Reply:
> I am using the term "prerequisite tasks" to avoid confusion with ...

Yes, but tasks don't get "satisfied", prerequisites do. Oliver's suggestion would probably be better.